airflow/triggers/external_task.py

```diff
@@ -59,7 +58,7 @@ def __init__(
         self.execution_dates = execution_dates
         self.poll_interval = poll_interval
 
-    def serialize(self) -> tuple[str, dict[str, Any]]:
+    def serialize(self) -> tuple[str, dict[str, typing.Any]]:
```

Comment:
Why typing.Any instead of Any...just curious

Reply:
Previously it’s importing from typing twice and I just figured this is a good chance to clean it up